browser/resources/brave_new_tab_page_refresh/components/app.tsx

```diff
@@ -20,6 +20,16 @@ export function App() {
   const [settingsView, setSettingsView] =
     React.useState<SettingsView | null>(null)
 
+  React.useEffect(() => {
+    let params = new URLSearchParams(location.search)
+    let settingsArg = params.get('openSettings')
+    if (settingsArg === null) {
+      return
+    }
+    setSettingsView('background')
```

Comment:
okay, so if we specify `openSettings` it will always open the `background` tab? Could we make it so that we can open any tab in settings (i.e. news) via the same mechanism?

Reply:
Yes - the background tab is the fallback. We'll also need to support a value of `BraveNews` which opens the news settings panel (when that is added).

> Could we make it so that we can open any tab in settings (i.e. news) via the same mechanism?

The current NTP does that [here](https://github.com/brave/brave-core/blob/7a2beb912139d6246081c90187870a9ba8e435d9/components/brave_new_tab_ui/containers/newTab/index.tsx#L305); it's currently only used with `openSettings=1` and `openSettings=BraveNews`. Do we want to stick with this `openSettings` URL param way of accessing the settings dialog?